mycroft/util/network_utils.py

```diff
@@ -6,6 +6,12 @@
 from .log import LOG
 
 
+def get_network_tests_config():
```

Comment:
Maybe a docstring?

Reply:
Indeed - I did the docstring as an explanation of the function, and a separate inline comment re wrapping to avoid circular import. 

If anyone has strong opinions that these should both be in the docstring I'm happy to amend.